src/timers.cc

```diff
@@ -13,49 +13,43 @@
 // limitations under the License.
 
 #include "timers.h"
-#include "internal_macros.h"
+
+#include <cstdlib>
+#include <ctime>
+#include <iostream>
+
+#include "internal_macros.h"  // IWYU pragma: keep
 
 #ifdef BENCHMARK_OS_WINDOWS
 #include <shlwapi.h>
 #undef StrCat  // Don't let StrCat in string_util.h be renamed to lstrcatA
 #include <versionhelpers.h>
 #include <windows.h>
-#else
-#include <fcntl.h>
-#ifndef BENCHMARK_OS_FUCHSIA
```

Comment:
you've lost these FUCHSIA checks.

i'd strongly advise against messing with the OS_ checks at all. they're prone to breakage.